bot/exts/moderation/infraction/management.py

```diff
@@ -311,6 +312,10 @@ async def search_user(self, ctx: Context, user: MemberOrUser | discord.Object) -
         )
         # Manually form mention from ID as discord.Object doesn't have a `.mention` attr
         prefix = f"<@{user.id}> - {user.id}"
+        # If the user has alts show in the prefix
```

Comment:
Just asking here, currently if the user has no infractions at all but has alt accounts, it won't show anything. 

It might not be intuitive at first to call the `alt info` command as the first reflex is to call the `i s` one.

Reply:
I'll add this info to the user command when ran in mod channels